yaks-html/lib/yaks/format/html.rb

```diff
@@ -39,8 +39,8 @@ def render_resource(resource, templ = section('resource'))
           .replace('.forms') {|div| render_forms(resource.forms).call(div) }
           .replace('.subresource') {|sub_templ| render_subresources(resource, templ, sub_templ) }
       end
-      alias render_collection_resource render_resource
-      alias render_null_resource render_resource
+      alias_method :render_collection_resource, :render_resource
+      alias_method :render_null_resource, :render_resource
```

Comment:
The order of arguments is the same as bash's `alias`, which is the opposite of `ln`, which follows the logic of `cp`. How much easier can it be? :smile: :smile: 

I'm cool with `alias_method`. I'd just really like someone to explain the difference to me, instead of cargo culting it.